RELEASE.md

```diff
@@ -0,0 +1,92 @@
+
+### Changelogs
+Boxo loosely follows the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. New commits should add changelog entries into the `[Unreleased]` section so that there is no changelog scramble when a new version needs to be released.
+
+We intend to use the standardized changelog to automate releases by onboarding with [Changelog Driven Release](https://github.com/pl-strflt/changelog-driven-release) (see [#269](https://github.com/ipfs/boxo/issues/269)).
+
+### Related Work
+Below are links of related/adjacent work that has informed some of the decisions in this document:
+1. https://github.com/ipfs/boxo/issues/170
+2. https://pl-strflt.notion.site/Kubo-Release-Process-5a5d066264704009a28a79cff93062c4
+3. https://github.com/ipfs/kubo/blob/master/docs/RELEASE_ISSUE_TEMPLATE.md
+
+## Release Process
+1. Create an issue for the version: https://github.com/ipfs/boxo/issues/new?title=Release+X.Y.Z (can be blank for now)
+2. Pin the issue
+3. Copy-paste the following checklist into the description:
+
+- [ ] Verify your [GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification) is configured in local git and GitHub
+- [ ] Ensure Boxo and Kubo are checked out on your system
+- [ ] Create a release branch called `release-vX.Y.Z` and push it
+  * Use `master` as base if `Z == 0`
+  * Use `release` as base if `Z > 0`
+- [ ] Tidy the changelog and push any changelog changes to `release-vX.Y.Z`
+- [ ] Create a draft PR merging `release-vX.Y.Z` into `release`
+  - [ ] Name it "Release vX.Y.Z"
+  - [ ] Reference this release issue
+- [ ] If `Z > 0`, then cherry-pick the necessary commits from `master` into `release-vX.Y.Z` using `git cherry-pick -x <commit>`
+- [ ] Ensure Boxo tests are passing
+- [ ] Ensure Kubo tests are passing
+  - [ ] Go to Kubo dir and run `go get github.com/ipfs/boxo@<commit_hash>` using the commit hash of the `release-vX.Y.Z` branch
+  - [ ] Run `go mod tidy` in repo root and in `docs/examples/kubo-as-a-library`
+  - [ ] Commit the changes and open a draft PR in Kubo
+  - [ ] Name the PR "Upgrade to Boxo vX.Y.Z"
+  - [ ] Paste a link to the Kubo PR in the Boxo PR, so reviewers can verify the Kubo test run
+  - [ ] Verify the CI passes
+- [ ] Add a commit in `release-vX.Y.Z` bumping the version in `version.json` to `vX.Y.Z`
+- [ ] Add a "release" label to the Boxo PR
+- [ ] After the release checker creates a draft release, copy-paste the changelog into the draft
+- [ ] Wait for approval from another Boxo maintainer
+- [ ] Merge the PR into `release`, _using "Create a Merge Commit"_, and do not delete the `release-vX.Y.X` branch
+  - [ ] Verify the tag is created
+- [ ] Announce the release
+  - [ ] Create an [IPFS Discourse](https://discuss.ipfs.tech) topic
+    - [ ] use `Boxo vX.Y.Z is out!` as the title
+    - [ ] use `boxo` as the topic
```

Comment:
boxo as the tag?
news as the category?
<img width="712" alt="image" src="https://user-images.githubusercontent.com/85411/231374530-f028237b-fdad-4ff7-9cc1-9efaf469be50.png">